### ISSUE TITLE
Possible for collection quest items to get into empty or non-numeric state

### ISSUE THREAD

**Issue by SabreCat** [HabitRPG/habitica#12360]
[//]: # (Before logging this issue, please post to the Report a Bug guild from the Habitica website's Help menu. Most bugs can be handled quickly there. If a GitHub issue is needed, you will be advised of that by a moderator or staff member -- a player with a dark blue or purple name. It is recommended that you don't create a new issue unless advised to.)

[//]: # (Bugs in the mobile apps can also be reported there.)

[//]: # (If you have a feature request, use "Help > Request a Feature", not GitHub or the Report a Bug guild.)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitica/issues/2760)

[//]: # (Fill out relevant information - UUID is found from the Habitia website at User Icon > Settings > API)
### General Info
  * UUID: 6f122835-1389-48a5-b27a-5fbafdc27beb
  * Browser: N/A
  * OS: iOS

### Description

User reported seeing strange quest update messages in party chat:

![IMG_7934](https://user-images.githubusercontent.com/4501321/86410272-1a38fe80-bc80-11ea-849b-a1118a5695d4.jpg)

Apparently, it's possible for this block of code

https://github.com/HabitRPG/habitica/blob/develop/website/server/models/group.js#L1133-L1184

to resolve knowing it should output a collection quest update, but not actually finding a proper collection item to dispense 0 of. Possibly the quest progress object can get into an unexpected state when immediately convening the same quest right after it completes, in a solo party?

#### Console Errors
[//]: # (Include any JavaScript console errors here.)

No relevant errors found in logging.

**Comment by SabreCat** [comment]
Setting this to "priority: minor" supposing it's a rare fluke; if we see this happening more frequently than this one-off, then the priority should definitely go up.

**Comment by AnhuaW** [comment]
@SabreCat Hi, may I take on this issue?

**Comment by CuriousMagpie** [comment]
Not a problem @AnhuaW! Thanks so much!

**Comment by AnhuaW** [comment]
Hi, I was looking at the blacksmith guidebook and trying to figure out a way to reproduce this bug. I could not seem to find the debug panel on my local installation. Should I try to connect to a test account before looking for the debug menu?

**Comment by CuriousMagpie** [comment]
@AnhuaW Yes, when you run your local install, you need to create at least one (probably more) users. Once you're logged in, you will find the debug menu on the bottom left of your screen, below the footer. It expands in a downwards direction. I hope this helped!

**Comment by AnhuaW** [comment]
> @AnhuaW Yes, when you run your local install, you need to create at least one (probably more) users. Once you're logged in, you will find the debug menu on the bottom left of your screen, below the footer. It expands in a downwards direction. I hope this helped!

Thanks!

**Comment by imnuvi** [comment]
Hi folks, is this issue still in progress? If not I would like to take this up

**Comment by AnhuaW** [comment]
> Hi folks, is this issue still in progress? If not I would like to take this up

I'm no longer working on the issue. I think you can ask the members to assign it to you :)

**Comment by imnuvi** [comment]
Hi @SabreCat  @CuriousMagpie can any of you let me know if I can take up this issue?

**Comment by joaoaspimienta** [comment]
@SabreCat @CuriousMagpie is no work being done on the issues? I'd like to contribute to Habitica but in the issues I checked users are asking for their contributions to be reviewed or approved and they're not getting any replies

**Comment by CuriousMagpie** [comment]
@joaoaspimienta, if you'd like to pick up this issue, please do so!
